pipelines/HSC/RC2.yaml

```diff
@@ -0,0 +1,379 @@
+description: The DRP pipeline specialized for the HSC RC2 dataset.
+instrument: lsst.obs.subaru.HyperSuprimeCam
+imports:
+  - $DRP_PIPE_DIR/ingredients/DRP.yaml
+  - $FARO_DIR/pipelines/metrics_pipeline_jointcal_fgcm.yaml
+tasks:
+  skyCorr:
+    class: lsst.pipe.drivers.skyCorrection.SkyCorrectionTask
+  jointcal:
+    # Running jointcal is something we want to do for all real-world
+    # instruments, so it doesn't belong in an HSC-specific pipeline, let alone
+    # an RC2-specific one.  We just haven't done the pipeline refactoring that
+    # would fix this, and it's not a huge loss because we haven't actually
+    # gotten jointcal running in production on anything other than HSC (it
+    # might just work, but we certainly can't claim it's supported).
+    class: lsst.jointcal.JointcalTask
+    config:
+      # Disable photometry because we don't need it (FGCM takes care of that).
+      # It would be better to disable this via config (DM-29885), but that's
+      # more refactoring we haven't yet done.
+      doPhotometry: false
+      python: |
```

Comment:
Note: this entire jointcal config block, except for the `doPhotometry` line just above can go away once I merge DM-29008 (which should happen today).

Reply:
After removing this block (I also removed the `doPhotometry` line, because of DM-29885), the expanded config from the pipeline on this branch differs from the expansion of `obs_subaru` `main` pipeline in two ways:

 - `main` has `config.colorterms.data` populated, while this branch does not.
 - `main` has `config.photometryRefObjLoader.filterMap` populated, while this branch doe not.

I'm fairly certain those don't matter as long as `doPhotometry` is `False`, but if they're necessary to make `doPhotometry=True` work on HSC, it seems appropriate to include them so turning on photometry just involves setting `doPhotometry=True`, and not populating these dicts.  Or is it no longer necessary to populated them anyway, due to the `anyFilterMapsToThis` stuff?  Is there any reason to still include them in the defaults, if that's the case?

(also CC @laurenam)

Reply:
Ah, good point! I removed the jointcal photometry filterMap from obs_subaru, but you're correct that we want it so there's just a single step to enable jointcal photometry fitting for HSC. I think this is a case where it's more appropriate to have the config it in the obs package, since you want it whether you're running the DRP pipeline or just jointcal itself.

Reply:
Yeah, I think that makes sense.  I’m a bit torn as I’m not a fan of having configs set that aren’t being used, but I see where having these set could save some pain for anyone wanting to turn photometry on in `jointcal`.  I suppose a caveat is that there needs to be colorterms appropriate for the refcat used for the photometry, but it looks like the default for `jointcal` is PS1, so all is good there.

Reply:
Ok, I've pushed a commit to the `obs_subaru` branch of this ticket that puts the `colorterms.py` and `filterMap.py` config loads back into the `jointcal.py`.  I've also set
```
config.applyColorTerms = True
```
there, which is actually different from the current HSC pipeline on `main` (i.e. on `main`, `applyColorTerms=False` for jointcal in Gen3).  Can either of you confirm that this is a no-op when photometry is not being run, and not a problem?

Reply:
Looks like you’ll just get a warning:
https://github.com/lsst/jointcal/blob/main/python/lsst/jointcal/jointcal.py#L589-L592

Reply:
Correct: jointcal does nothing with colorterms when `doPhotometry=False`.

Reply:
Sounds like we should probably remove that warning, but I'll leave the pipeline here as it is, then.